wormhole-connect/src/hooks/useRoutesQuotesBulk.ts

```diff
@@ -0,0 +1,150 @@
+import { useState, useEffect, useMemo } from 'react';
+import { Chain, amount as sdkAmount } from '@wormhole-foundation/sdk';
+
+import { getTokenDecimals } from 'utils';
+import config from 'config';
+import { toDecimals } from 'utils/balance';
+
+type RoutesQuotesBulkParams = {
+  sourceChain: Chain | undefined;
+  sourceToken: string;
+  destChain: Chain | undefined;
+  destToken: string;
+  amount: number;
+  nativeGas: number;
+};
+
+export type RouteQuote = {
+  error: string;
```

Comment:
It would be clearer to make `error` optional, and keep it `undefined` in the case where there is no error.
```suggestion
  error?: string;
```